openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -463,6 +463,17 @@ const EntityTable = ({
     );
   };
 
+  /* eslint-disable-next-line */
```

Comment:
Can we remove this?

Reply:
As we are going to remove the react-table package, this will be taken care of